docs/admin/installation/hardware.rst

```diff
@@ -474,11 +500,10 @@ Server we recommend. To do so, select the "Boot and Install with the HWE
 Kernel" option in the boot menu for Ubuntu Server.
 
 .. _nuc11_recommendation:
-.. _intel-11th-gen-nuc:
 
-11th-gen NUC
-~~~~~~~~~~~~
-We have tested and can recommend the `NUC11PAHi3 <https://www.asus.com/us/displays-desktops/nucs/nuc-kits/nuc-11-performance-kit/techspec/>`__.
+Intel 11th-gen NUC
+~~~~~~~~~~~~~~~~~~
+We have tested and can recommend the `Intel NUC11PAHi3 <https://ark.intel.com/content/www/us/en/ark/products/205033/intel-nuc-11-performance-kit-nuc11pahi3.html>`__.
```

Comment:
The intel link is a 404, while the removed ASUS link is fine.

Reply:
Switched back to ASUS link.